resqpy/rq_import/_add_surfaces.py

```diff
@@ -34,21 +35,25 @@ def add_surfaces(
         quad_triangles (bool, default False): if True, 4 triangles per quadrangle will be used for mesh formats, otherwise 2
         surface_file_list (list, default None): list of full file names (paths), each holding one surface
         make_horizon_interpretations_and_features (bool, default True): if True, feature and interpretation objects are created
+        interpretation_type (str, default 'horizon'): if 'make_horizon_interpretations_and_features' is True, feature and interpretation objects are added. Default is 'horizon', other options are 'fault' and 'geobody'
 
     Returns:
         resqml model object with added surfaces
+
+    Note: where interpretation_type is 'fault' the faultinterpretation added will be defined as a normal fault.
```

Comment:
Let's fix this too, with a new bool argument fault_is_normal, defaulting to True, and only used for faults.